now.json

```diff
@@ -1,10 +1,60 @@
 {
-    "name": "magento-venia",
-    "alias": ["magento-venia", "veniapwa.com"],
-    "engines": {
-        "node": "10.x.x"
+    "version": 2,
+    "name": "venia",
+    "alias": [
+        "magento-venia-develop"
+    ],
+    "scope": "magento-research",
+    "env": {
+        "UPWARD_JS_BIND_LOCAL": "false",
+        "NOW_DEBUG": "1",
+        "MAGENTO_BACKEND_URL": "https://release-dev-231-npzdaky-zddsyhrdimyra.us-4.magentosite.cloud/",
+        "DEBUG": "upward-js:*"
     },
-    "public": true,
-    "version": 1,
-    "type": "npm"
+    "builds": [
+        {
+            "src": "packages/venia-concept/package.json",
+            "use": "@now/static-build"
+        },
+        {
+            "config": {
+                "includeFiles": [
+                    ".env.dist",
+                    "venia-upward.yml",
+                    "static/*",
+                    "templates/*"
+                ],
+                "maxLambdaSize": "50mb"
```

Comment:
😰

Reply:
Removing sharp dropped this to 3mb soooo *shrug*

Reply:
As we saw, removing the image opt middleware lowered this size so you *should* be able to remove this and have it set back to the default of `5mb`.

https://zeit.co/docs/v2/deployments/concepts/lambdas/#maximum-bundle-size